CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 17.0.0
+
+- Removed: `stylelint` less than `16.23.0` from peer dependencies.
+- Added: `media-type-no-deprecated` rule.
+- Added: `nesting-selector-no-missing-scoping-root` rule.
```

Comment:
Is it too late to put this in standard?
[As I explained](https://github.com/stylelint/stylelint/issues/8406#issuecomment-3078779615), it doesn't belong in recommended.